app/nexus/internal/net/cache/net.go

```diff
@@ -116,6 +117,11 @@ func FetchFromCache(source *workloadapi.X509Source) (string, error) {
 	}
 
 	data, err := net.Post(client, api.UrlKeeperRead(), md)
+	if err != nil {
+		return "", fmt.Errorf(
+			"FetchFromCache: failed to post request: %w", err,
```

Comment:
I see that other errors in this function don't propagate using error wrapping - is this intentional? If so, I can remove the wrapping from this error here.

Reply:
My preference would be to wrap the error if we are pushing the error to the upper stack.

At least, that's the convention in the majority of the codebase.